src/ProductConstructionService/ProductConstructionService.Api/PcsStartup.cs

```diff
@@ -312,6 +319,8 @@ public static void ConfigureApi(this IApplicationBuilder app, bool isDevelopment
             {
                 controllers.AllowAnonymous();
             }
+            
+            e.MapGitHubWebhooks(path: GitHubWebHooksPath, secret: webHooksSecret);
```

Comment:
What does this do without a secret?

Reply:
I am asking because when I was rewriting the `Program.cs`, I mostly removed all access to configuration there and kept it in `PcsStartup.cs`

Reply:
Oh, I see some leaked back in the change with the dedicated VMR queue. But mostly I just kept it in the PcsStartup and I kept the ConfigurationKey private so it's super obvious we don't use it anywhere but here. THen it's very easy to figure out where is config read from and what is expected inside

Reply:
Ah I see, I did it because the `IApplicationBuilder` didn't have access to the `Configuration`, but we can just pass the `Configuration` in this case.

Reply:
Alternatively you could also get the configuration from the app:
`app.ApplicationServices.GetRequiredService<ConfigurationManager>()`

Reply:
I like that better, I didn't realize it had the ServiceProvider, since it's under a different name, it didn't come up when I was looking for it